src/main/java/player/Computer.java

```diff
@@ -0,0 +1,26 @@
+package player;
+
+import move.Move;
+import move.MoveFactory;
+import move.Moves;
+
+import java.util.List;
+import java.util.Random;
+
+public class Computer implements Player {
+
+    private final MoveFactory moveFactory;
+    private String name;
+
+    public Computer() {
+        this.name = "computer";
+        moveFactory = new MoveFactory();
+    }
+
+    public Move makeMove() {
+        List<String> moves = Moves.possibleMoves;
+        String randomMove = moves.get(new Random().nextInt(moves.size()));
```

Comment:
This class is hard to test because it has non-deterministic behaviour inside it.  🕵️‍♀️ Think about how to extract the uncertainty away. 🔎  Perhaps look up random seeds . 🔎  Perhaps think about dependency injection.